test/kitchen/site-cookbooks/dd-system-probe-check/recipes/default.rb

```diff
@@ -63,6 +73,7 @@
 execute 'disable firewalld on redhat' do
   command "systemctl disable --now firewalld"
   user "root"
+  ignore_failure true
```

Comment:
Which platforms does this fail on?

Reply:
`arm64` weirdly. I tried both the `service` and `systemd_unit` Chef resources, but neither of them worked.

Reply:
Will this be logged if it fails? Does that route lookup test still pass on `arm64`?

Reply:
>Will this be logged if it fails? 

It does log if it fails. You have to use `:quiet` to suppress the logs.

> Does that route lookup test still pass on `arm64`?

Everything is passing 😁